src/components/store/SizeSelector/index.tsx

```diff
@@ -11,21 +11,23 @@ interface SizeSelectorProps {
 const SizeSelector = ({ currSize, options, onSizeChange }: SizeSelectorProps) => {
   const myID = useId();
```

Comment:
This comes from Andy's PR, but IDs are used so that inputs and labels can be flattened (`<input id={id} /> <label htmlFor={id}>...</label>` instead of `<label><input /></label>`). This allows the CSS to do `input:checked + label` etc and make `label` (which is not a replaced element so it's easier to style) to be what the user sees as the radio button, whilst still preserving built-in browser functionality.

I suppose we could use `label:has(:checked)` now, but that CSS feature feels uncomfortably new

We can use JavaScript to track the checked state of each radio button, but it's a lot more code than just relying on CSS.